spec/annexes/extension_schema.adoc

```diff
@@ -298,12 +309,12 @@ If the `gpkg_data_column_constraints` table contains rows with `constraint_type`
 [cols="1,5a"]
 |========================================
 |*Test Case ID* |+/extensions/schema/data_columns/table_name+
-|*Test Purpose* |Verify that for each gpkg_data_columns row, the table_name value matches a row in gpkg_contents.
+|*Test Purpose* |Verify that for each gpkg_data_columns row, the table_name value matches a row in gpkg_contents or gpkg_extensions.
```

Comment:
This should be "... `gpkg_contents` or `gpkg_extensions`." That is, back-tick quote the names.

Reply:
Probably, but I'm not that excited about adding the hundred '`' marks needed to make the entire ATS right.